containers/ffmpeg-api/k8s.yaml

```diff
@@ -73,41 +73,41 @@ spec:
             memory: 16Gi #64Gi
             ephemeral-storage: 200Gi
           limits:
-            cpu: 4 #16
+            # cpu: 4 #16
             memory: 16Gi #64Gi
             ephemeral-storage: 200Gi
-        volumeMounts:
-        - name: gcs-fuse-csi-gke-input
-          mountPath: /input
-        - name: gcs-fuse-csi-gke-event-input
-          mountPath: /input-event
-        - name: gcs-fuse-csi-output
-          mountPath: /output
-      volumes:
-      - name: gcs-fuse-csi-gke-input
-        csi:
-          driver: gcsfuse.csi.storage.gke.io
-          volumeAttributes:
-            bucketName: gcs-testtest-431919-gcp-gke-input
-            fileCacheCapacity: "100Gi"
-            cache-dir: "/var/cache"
-            enable-parallel-downloads: "true"
-      - name: gcs-fuse-csi-gke-event-input
-        csi:
-          driver: gcsfuse.csi.storage.gke.io
-          volumeAttributes:
-            bucketName: gcs-testtest-431919-gcp-gke-input
-            fileCacheCapacity: "100Gi"
-            cache-dir: "/var/cache"
-            enable-parallel-downloads: "true"
-      - name: gcs-fuse-csi-output
-        csi:
-          driver: gcsfuse.csi.storage.gke.io
-          volumeAttributes:
-            bucketName: gcs-testtest-431919-gcp-test-output
-            fileCacheCapacity: "100Gi"
-            cache-dir: "/var/cache"
-            enable-parallel-downloads: "true"
+      #   volumeMounts:
+      #   - name: gcs-fuse-csi-gke-input
+      #     mountPath: /input
+      #   - name: gcs-fuse-csi-gke-event-input
+      #     mountPath: /input-event
+      #   - name: gcs-fuse-csi-output
+      #     mountPath: /output
+      # volumes:
+      # - name: gcs-fuse-csi-gke-input
+      #   csi:
+      #     driver: gcsfuse.csi.storage.gke.io
+      #     volumeAttributes:
+      #       bucketName: gcs-__PROJECT_ID__-__CUSTOMER_ID__-test-input
+      #       fileCacheCapacity: "100Gi"
+      #       cache-dir: "/var/cache"
+      #       enable-parallel-downloads: "true"
+      # - name: gcs-fuse-csi-gke-event-input
+      #   csi:
+      #     driver: gcsfuse.csi.storage.gke.io
+      #     volumeAttributes:
+      #       bucketName: gcs-__PROJECT_ID__-__CUSTOMER_ID__-test-input
+      #       fileCacheCapacity: "100Gi"
+      #       cache-dir: "/var/cache"
+      #       enable-parallel-downloads: "true"
+      # - name: gcs-fuse-csi-output
+        # csi:
+        #   driver: gcsfuse.csi.storage.gke.io
+        #   volumeAttributes:
+        #     bucketName: gcs-__PROJECT_ID__-__CUSTOMER_ID__-test-output
+        #     fileCacheCapacity: "100Gi"
+        #     cache-dir: "/var/cache"
+        #     enable-parallel-downloads: "true"
```

Comment:
Probably good to delete this whole commented out section unless you really want it here for people to see?

Reply:
I will delete, was going to revisit. But was having issues with integrating GCS fuse in differentt Namespaces in the same cluster I think because it was trying to mount the same bucket.